lib/ddtrace/contrib/grape/endpoint.rb

```diff
@@ -196,6 +203,17 @@ def analytics_sample_rate
             datadog_configuration[:analytics_sample_rate]
           end
 
+          def exception_is_error?(exception)
+            matcher = datadog_configuration[:error_statuses]
+            return false unless exception
+            if exception.respond_to?('status') && matcher.include?(exception.status)
+              status = exception.status
+            else
+              return true
+            end
+            !status.nil?
+          end
```

Comment:
Given that `datadog_configuration[:error_statuses]` willl now default to nil, we can clean up this logic a bit. currently, by having `if exception.respond_to?('status') && matcher.include?(exception.status)` in the same `if` statment, we aren't properly returning `false` when `status` exists but `matcher.include?(exception.status)` returns false. The below updates to account for this
```suggestion
          def exception_is_error?(exception)
            matcher = datadog_configuration[:error_statuses]
            return false unless exception
            return true unless matcher
            return true unless exception.respond_to?('status') 
            matcher.include?(exception.status)
          end
```